thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -130,4 +130,7 @@ struct MediaAtom {
 
   /** suppress related content of the (optional) Composer page **/
   21: optional bool suppressRelatedContent
+
+  /** alt text for Composer page trail image**/
+  22: optional string altText
```

Comment:
Why not add this field to `Image` instead, and then bubble it up via `trailImage` when displaying the atom?